src/utility/sigmadao.utility.ts

```diff
@@ -18,3 +20,20 @@ export const redirectTo = (router: Router, path: EndPoint) => {
 export const calculatePercentage = (value: number, total: number): number => {
 	return total > 0 ? (value / total) * 100 : 0.0;
 };
+
+export const validateFundAmount = (
+	_rule: Rule,
+	value: string,
+	formValue: number | undefined
```

Comment:
```suggestion
	formValue?: number
```
This should serve the same purpose. Please check